src/game_api/rpc.cpp

```diff
@@ -40,19 +40,19 @@
 #include "thread_utils.hpp"     // for OnHeapPointer
 #include "virtual_table.hpp"    // for get_virtual_function_address, VIRT_FUNC
 
-uint32_t setflag(uint32_t flags, int bit) // shouldn't we change those to #define ?
+inline uint32_t setflag(uint32_t flags, int bit) // shouldn't we change those to #define ?

```

Comment:
Please don't put inline on functions unless you actually need inline linkage.

Reply:
What's wrong with inline?

Reply:
`inline` declares a function as having `inline` linkage, it does not tell the compiler to inline the function. And you don't need `inline` linkage here from what I can see so it's completely unnecessary.